core/src/main/java/org/frankframework/pipes/IsUserInRolePipe.java

```diff
@@ -99,7 +115,7 @@ public String getRole() {
 		return role;
 	}
 
-	/** the j2ee role to check.  */
+	/** the j2ee role(s) to check, if multiple roles specified, the first matching role is prioritized.  */
```

Comment:
With `is prioritized` you mean wins?

Reply:
Exactly.

The Frank developer could specify `role1,role2,role3`, if the user in all roles, the first specified role will be matched.

Reply:
Clear, please adapt the wording a bit, so it is clear for others as well. Thanks 👍🏻